src/rascal/representations/calculator_spherical_expansion.hh

```diff
@@ -1082,8 +1099,10 @@ namespace rascal {
             radial_contribution_hypers.at("optimization").template get<json>();
 
         double accuracy{this->get_interpolator_accuracy(optimization_hypers)};
-        double range_begin{this->get_range_begin(optimization_hypers)};
-        double range_end{this->get_range_end(optimization_hypers)};
+        // minimal distance such that it is still stable with the interpolated
+        // function
+        double range_begin{math::SPHERICAL_BESSEL_FUNCTION_FTOL};
+        double range_end{this->interaction_cutoff};
```

Comment:
The reason to keep this as a user input was to allow for a NL with `r_c > this->interaction_cutoff` so one could set `range_end` to `r_c`. At the moment it is not really used so I guess it is fine to remove it from the user inputs

Reply:
could you introduce a case where this might happen in the future? I have a hard time to imagine when this would be the case

Reply:
it is fine as is, I am just mentioning the reason why it was like that in the first place.